config/initializers/setup_mail.rb

```diff
@@ -0,0 +1,12 @@
+if Rails.env.development?
```

Comment:
I'm confused by your note. Do you mean that I should also have once of these setup blocks for production?